apis/apis.json

```diff
@@ -145,6 +145,23 @@
     }
   },
 
+  {
+    "id": "Google.Cloud.Billing.V1",
+    "generator": "micro",
+    "protoPath": "google/cloud/billing/v1",
+    "productName": "Google Cloud Billing API",
+    "productUrl": "https://cloud.google.com/billing/docs/",
+    "version": "1.0.0-beta00",
+    "type": "grpc",
+    "description": "Recommended Google client library to access the Google Cloud Billing API, which define a budget plan and the rules to execute as spend is tracked against that plan.",
```

Comment:
nit: which define**s**

Reply:
Whoops. Will probably make it "which allows you to define" or similar. Will check the discovery doc again.